src/foam/core/AbstractFObject.js

```diff
@@ -107,31 +107,6 @@ foam.CLASS({
   // ],
 
   methods: [
-    {
```

Comment:
I would rather leave this here. For Android clients code size is more important than performance so we might like to disable creating the specific methods in that case.

Reply:
but this will create reference to compare is ambiguous